README.md

```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
   [Read more...](doc/error/e-unsupported.md)
 
 
+## Contribution
+
+If you are having trouble using Globalize after reading the documentation
+carefully, please post a question to [StackOverflow with the
+"javascript-globalize" tag][]. Questions that include a minimal demo are more
+likely to receive responses.
+
+In the spirit of open source software development, we always encourage
+community code contribution. To help you get started and before you jump into
+writing code, be sure to read [Writing Code for jQuery Foundation Projects][].
+
+[StackOverflow with the "javascript-globalize" tag]: http://stackoverflow.com/tags/javascript-globalize
+[Writing Code for jQuery Foundation Projects]: http://contribute.jquery.org/code/
+
+For ideas where to start contributing, find what you like best: [quick
+changes][], [new features][], [bug fixes][], or [documentation improvements][];
+or by modules: [date module][], [currency module][], [message module][], [number
+module][], [plural module][], or [relative time module][].
+
+[bug fixes]: https://github.com/jquery/globalize/labels/bug
+[documentation improvements]: https://github.com/jquery/globalize/labels/docs
+[new features]: https://github.com/jquery/globalize/labels/new%20feature
+[quick changes]: https://github.com/jquery/globalize/labels/quick%20changes
+
+[currency module]: https://github.com/jquery/globalize/labels/currency%20module
+[date module]: https://github.com/jquery/globalize/labels/date%20module
+[message module]: https://github.com/jquery/globalize/labels/message%20module
+[number module]: https://github.com/jquery/globalize/labels/number%20module
+[plural module]: https://github.com/jquery/globalize/labels/plural%20module
+[relative time module]: https://github.com/jquery/globalize/labels/relative%20time%20module
+
+### Roadmap
+
+To avoid manual maintenance we avoid duplicating information elsewhere, so our
```

Comment:
I'd drop the disclaimer, just explain what it is, about the "avoid duplicating" part.